charts/kube-prometheus-stack/templates/prometheus/additionalScrapeConfigs.yaml

```diff
@@ -12,5 +12,9 @@ metadata:
     app: {{ template "kube-prometheus-stack.name" . }}-prometheus-scrape-confg
 {{ include "kube-prometheus-stack.labels" . | indent 4 }}
 data:
-  additional-scrape-configs.yaml: {{ tpl (toYaml .Values.prometheus.prometheusSpec.additionalScrapeConfigs) $ | b64enc | quote }}
```

Comment:
the posssibility to use string ot tpl should be reflected in some info in the values.yaml too.

Reply:
Thanks for reviewing,  `values.yaml` updated. 
If you have ideas what can be improved in PR, please let me know.